systemvm/debian/opt/cloud/bin/cs/CsDhcp.py

```diff
@@ -196,6 +203,9 @@ def add(self, entry):
             self.dhcp_opts.add("%s,%s" % (tag, 3))
             self.dhcp_opts.add("%s,%s" % (tag, 6))
             self.dhcp_opts.add("%s,%s" % (tag, 15))
+            self.dhcp_leases.search(entry['mac_address'], "0 %s %s %s *" % (entry['mac_address'],
```

Comment:
same as above - can you check if the lease should have a high expiry (80yrs or inf?) instead of `*`, I'm not sure what 0 and `*` mean here

Reply:
0 is infinite, * is the client id (which is generally the mac, but can change)
Taken this format from existing leases file